### PR TITLE
feat: [SSCA-895] SSCA side nav changes to support ssca prefixed paths

### PR DESCRIPTION
**Pull request by udayvunnam**
#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
- MessageMetadata: `trigger messagecheck`
- GitLeaks: `trigger gitLeaks`
</details>

## PR check failures and solutions
https://harness.atlassian.net/wiki/spaces/BT/pages/21106884744/PR+Checks+-+Failures+and+Solutions

### Rebase Instructions:
To rebase your branch onto the latest changes in the target branch, simply use the following command in a comment: `/rebase`

### CODE REVIEW

File: src/modules/75-ssca/RouteDestinations.tsx [hunk 1]
```diff
@@ -86,6 +84,19 @@ export default (
       />
     </RouteWithLayout>
 
+    {/* no exact, to match any sublevel of artifacts which can be defined within MFE
+    also not that wildcard pattern of /account/:accountId/:module(ssca) is matched for pipeline routes */}
```
Comment (udayvunnam) [inline src/modules/75-ssca/RouteDestinations.tsx:88]: FYI

File: src/modules/75-ssca/components/SSCASideNav.tsx [hunk 2]
```diff
@@ -17,6 +16,7 @@ import { useAppStore } from 'framework/AppStore/AppStoreContext'
 import { useStrings } from 'framework/strings'
 import ProjectSetupMenu from '@common/navigation/ProjectSetupMenu/ProjectSetupMenu'
 import { FeatureFlag } from '@common/featureFlags'
+import { useFeatureFlag } from '@modules/10-common/hooks/useFeatureFlag'
```
Comment (udayvunnam) [inline src/modules/75-ssca/components/SSCASideNav.tsx:19]: use FF from common instead of direct SDK

**Review by rameshx — DISMISSED** [review]

**Comment by udayvunnam** [comment]
retrigger titlecheck

**Comment by udayvunnam** [comment]
trigger messagecheck

**Comment by udayvunnam** [comment]
retrigger messagecheck

**Review by pankajsingh-harness — DISMISSED** [review]

**Review by chetanvekariya-harness — APPROVED** [review]

**Review by rameshx — APPROVED** [review]

**Review by chetanvekariya-harness — APPROVED** [review]

**Pull request merged**